### DISCUSSION TITLE
Bookmark File for later use

### DISCUSSION THREAD

**Discussion by SevenSamuraiSoldier** [PaulWoitaschek/Voice#1585] [Ideas]
Please implement a feature to create local bookmark file (with time stamps & name etc.) to be stored in the respective audiobook file/folder path (Similar to Smart audiobook player) so that users can refer the file for later use. Users can also send the bookmarks file to other people so that they can have the chapter/table of content bookmarks already in place while listening.

A simple text file with keywords for name & time stamps for the app to recognize the content will be sufficient enough.

**Comment by trymeouteh** [comment]
Not to be confused with [Bookmark for saving were you left off](https://github.com/PaulWoitaschek/Voice/discussions/1346).

I would like this also. A file for each audiobook which will save the bookmark timestamp and bookmark name. This way you can save your favourite parts of a chapter or even takes notes by saving a bookmark in a chapter.

This data should be saved in a separate file which could be synced with other devices and backup separately from the audiobook file.

Maybe using subtitle files as a solution such as an SRT file?
https://fileinfo.com/extension/srt


Hopefully Cozy add this feature also
https://github.com/geigi/cozy/issues/180

**Comment by PostHumanDev** [comment]
Being able to export Notes/Bookmarks as a text file would improve the app usability as a study tool.
